randovania/gui/widgets/reporting_optout_widget.py

```diff
@@ -0,0 +1,39 @@
+from PySide6 import QtWidgets
+
+import randovania
+from randovania.gui.generated.reporting_optout_widget_ui import Ui_ReportingOptOutWidget
+from randovania.gui.lib import common_qt_lib, signal_handling
+from randovania.interface_common.options import Options
+
+
+class ReportingOptOutWidget(QtWidgets.QWidget, Ui_ReportingOptOutWidget):
+    options: Options
+
+    def __init__(self):
+        super().__init__()
+        self.setupUi(self)
+        common_qt_lib.set_default_window_icon(self)
+
+        if randovania.is_dev_version():
+            self.intro_label.setText(
+                self.intro_label.text() + "\n\nTo help with the beta testing process, "
+                                          "these settings are always enabled in dev builds."
```

Comment:
it may be wise to specifically state that by authenticating in a dev build, they are agreeing to the privacy policy and opting in to the monitoring

Reply:
I was planning on updating the message you react to in Discord.